src/content/en/updates/2019/08/barcode-detection.md

```diff
@@ -0,0 +1,128 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Barcode Detection API allows for barcodes to be detected in images.
+
+{# wf_updated_on: 2019-08-05 #}
+{# wf_published_on: 2019-08-01 #}
+{# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: The Barcode Detection API allows for barcodes to be detected in images. #}
+{# wf_blink_components: Blink>ImageCapture #}
+
+# Shipping the Barcode Detection Part of the Shape Detection API {: .page-title}
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+<div class="clearfix"></div>
+
+Success: We were working the specification for this API as part of the
+[capabilities project](/web/updates/capabilities). Following a successful Origin Trial,
+barcode detection has now launched.
+
+You may remember [my post from January](../01/shape-detection)
+on the [Shape Detection API](https://wicg.github.io/shape-detection-api),
+an API that defines three interfaces: `BarcodeDetector`, `FaceDetector`,
+and `TextDetector`. I'm now happy to share that following a successful Origin Trial,
+we're shipping the `BarcodeDetector` part of the Shape Detection API in Chrome&nbsp;78.
+The `FaceDetector` and the `TextDetector` interfaces still need some more fine-tuning.
```

Comment:
After a little bit of research, apparently this should be singular "interface" after all.

What I found is that if we say "The `FaceDetector` and `TextDetector` interfaces" then it's plural, but if we add a "the" in front of `TextDetector`, "interface" should be singular: "The `FaceDetector` and **the** `TextDetector` interface". 

What do you think?

Reply:
Singular it is :-)